components/Headers/PageHeader.js

```diff
@@ -145,28 +140,31 @@ const PageHeader = ({
                       <img className="w-6 h-6 mb-1" src="/Advanced.svg" />
                       Advanced View
                     </button>
-
-                    <Link href="/notes">
-                      <a
-                        href="#"
-                        className="border-transparent text-current flex flex-col items-center p-2 rounded  text-sm font-medium hover:bg-nav-hover dark:hover:bg-dark-bg"
-                      >
-                        <img className="w-6 h-6 mb-1" src="/notes.svg" />
-                        Notes
-                      </a>
-                    </Link>
-                    <Link href="/bookmark">
-                      <a
-                        href="#"
-                        className="border-transparent text-current flex flex-col items-center p-2 rounded border-b-2 text-sm font-medium hover:bg-nav-hover dark:hover:bg-dark-bg"
-                      >
-                        <img
-                          className="w-6 h-6 mb-1"
-                          src="/bookmark-header.svg"
-                        />
-                        Bookmark
-                      </a>
-                    </Link>
+                    {process.env.NODE_ENV !== "development" ? null : (
```

Comment:
Hi @nishant-922, codes looks good. Just wondering how about if we move this logic into a custom hook `useEnvironment` in such a way that we could pick env values as follows:

```jsx
const { isProduction, isDevelopment } = useEnvironment()
```

And then we can import these values inside a component and conditionally render the same. Let me know if you need further information on that, thanks

Reply:
Hello @redraushan , thank you for reviewing my code. yeah it is a great suggestion and makes our work very easy.